src/add/operator/zip-static.ts

```diff
@@ -1,5 +1,5 @@
 import {Observable} from '../../Observable';
 import {zip} from '../../operator/zip-static';
-Observable.zip = zip;
+Observable.zip = <any>zip;
```

Comment:
In this PR specific, I believe this was added to ensure to not to break build. Asking @david-driscoll to confirm once again.